fusionauth/resource_fusionauth_reactor.go

```diff
@@ -0,0 +1,122 @@
+package fusionauth
+
+import (
+	"context"
+
+	"github.com/FusionAuth/go-client/pkg/fusionauth"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func newReactor() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: createReactor,
+		ReadContext:   readReactor,
+		UpdateContext: updateReactor,
+		DeleteContext: deleteReactor,
+		Schema: map[string]*schema.Schema{
+			"license_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Sensitive:   true,
+				Description: "The license Id to activate",
+			},
+			"license": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Sensitive:   true,
+				Description: "The Base64 encoded license value. This value is necessary in an air gapped configuration where outbound network access is not available.",
+			},
+		},
+	}
+}
+
+func buildReactor(data *schema.ResourceData) fusionauth.ReactorRequest {
+	reactor := fusionauth.ReactorRequest{
+		LicenseId: data.Get("license_id").(string),
+		License:   data.Get("license").(string),
+	}
+	return reactor
+}
+
+func createReactor(_ context.Context, data *schema.ResourceData, i interface{}) diag.Diagnostics {
+	client := i.(Client)
+	reactor := buildReactor(data)
+
+	resp, faErrs, err := client.FAClient.ActivateReactor(reactor)
+	if err != nil {
+		return diag.Errorf("CreateReactor err: %v", err)
+	}
+
+	if err := checkResponse(resp.StatusCode, faErrs); err != nil {
+		return diag.FromErr(err)
+	}
+
+	// Every terraform resource needs an id. Since none of the fusion auth reactor apis return
+	// the id of the reactor, we will need to make one up. There can only be one reactor
+	// so this should be safe.
+	data.SetId("fusion-auth-reactor-id")
+
+	return nil
+}
+
+func updateReactor(_ context.Context, data *schema.ResourceData, i interface{}) diag.Diagnostics {
+	client := i.(Client)
+	reactor := buildReactor(data)
+
+	resp, faErrs, err := client.FAClient.ActivateReactor(fusionauth.ReactorRequest{
```

Comment:
Both the create and update use the `ActivateReactor` method. 

There really is no "create" endpoint for the reactor because it always exists, this endpoint sets the license data (for the first time or any subsequent times).